content/getting-started/first-steps.md

```diff
@@ -32,20 +34,19 @@ Not sure what modules are? Catch up by reading about them in [NestJS](https://do
 :::
 
 ```typescript title="discord.module.ts"
-import { Module } from '@nestjs/common';
-import { DiscordService } from './discord.service';
+import { Module } from "@nestjs/common";
```

Comment:
Also, why using double quotes? All project use linter with single quotes, can you fix, please?